src/pyhf_benchmark/run.py

```diff
@@ -135,13 +135,12 @@ def run(computation, backend, path, url, model_point, number, mode):
             meta = metas
             meta["backend"] = bk
             run_manager.start(meta)
-
-            number = int(number) if number.isdigit() else number
+            version = int(number) if number.isdigit() else number
```

Comment:
```suggestion
            interpolator_code = int(number) if number.isdigit() else number
```

`version` is kinda vague in meaning here, so I'd just be explicit with the naming.

Reply:
I will use another variable `interpolation_type` to replace `number`, otherwise it will cause a bug in for loop